### ISSUE TITLE
cancer variant databases for ingest

### ISSUE THREAD

**Issue by mellybelly** [NCATS-Tangerine/ncats-ingest#27]
see this nice list:
https://github.com/seandavi/awesome-cancer-variant-databases

related to #15, @dnahotline, @mbrush @stuppie @pnrobinson can you prioritize these and make a Dipper/Wikidata/BioThings plan? Would like most available for Monarch too, so good for joint ingest planning.

**Comment by pnrobinson** [comment]
I everybody -- I am starting to understand how Dipper works thanks to your help. I think it would be good to have a discussion of how we want to set up the model. There are a few issues with data ingest such as the fact that OncoKB just has gene symbol and protein-mutation (e.g., A423P), and we would prefer to have the genomic coordinates as well (and perhaps a preferred transcript for the mutations). This is addressable but will require some finessing. I am hoping to extend the python hgvs to be able to go from "p." to "c."; Reece said that going from "c." to "g." is now functioning well.
Another useful thing would be to add the cancer type to the OncoKB data by pulling it from the abstracts. I am wondering if it would be useful for us to get in contact with that group and propose they improve/extend their data model.

**Comment by stuppie** [comment]
For dgidb, see ticket here: https://github.com/monarch-initiative/dipper/issues/446
And direct communication with dgidb team here: https://github.com/griffithlab/dgi-db/issues/141 and https://github.com/griffithlab/dgi-db/issues/142
As a side note: DGIdb includes CIViC, DoCM, MyCancerGenome

**Comment by mbrush** [comment]
'Cancer variant databases' covers a broad and diverse domain.  The summary below attempts to tease out some of the different datatypes to consider in this space, and begins a list of data sources to consider/prioritize. Others feel free to add/modify as needed - I think you can directly edit my comment here if you’d like.  
 
### Data Types
**Primary Variant 'Associations'**
1. **Pathogenicity / Diagnostic** - describes a variant's causation of / correlation with disease (and therefore its diagnostic utility)
2. **Predictive** - describes a variant's impact on response to treatment for a particular disease
3. **Prognostic** - describes a variant’s impact on disease progression, severity, or patient survival
4. **Predisposition** - describes how a variant may confer susceptibility to disease
5. **Functional Effect** - describes impact of variant on protein function
6. **Drug Interaction** - describes molecular interactions between gene products and drugs
7. **Expression** - describes aspects of gene expression in different conditions
 
**Additional Variant 'Metadata'**
1.	**Variant identifiers** -  no universal system/authority here.  identifier usage is inconsistent and sparse.
     a. ClinVar and dbSNP are databases that provide variant identifiers, but many variants not registered here, and most other soruces do not reference these identifiers.
     b. Often a HGVS label is all we get, and often we only get protein level name (e.g. K99E)  that must be translated  to genomic variant name. 
     c. Efforts such as the [ClinGen Allele Registry](http://reg.clinicalgenome.org/site/cg-registry) and [GA4GH/VICC VMC model](https://docs.google.com/document/d/12E8WbQlvfZWk5NrxwLytmympPby6vsv60RxCeD5wc1E/edit#heading=h.qpi6nfi6nby6) are working on solutions we could potentially adopt/partner  with.
2.	**Variant position** - as noted, position is often bundled in with the variant's name or id (e.g. when HGVS is used). As metadata, position can be specified in different contexts (e.g. protein- vs -transcript - vs chromosome), which may need to be translated between.
3.	**Variant origin** - germline vs somatic  origin of the mutation
4.	**Association evidence/provenance** - sources, methods, and data used to support a variant association
 
### Cancer Variant Data Sources
1.	ClinVar - https://www.ncbi.nlm.nih.gov/clinvar/
2.	CIViC - https://civic.genome.wustl.edu/home
3.	Oncokb Precision Oncology Knowledge Base  - http://oncokb.org/#/DoCM
4.	Jackson Lab Clinical Knowledge Base - https://www.jax.org/clinical-genomics/clinical-offerings/ckb
5.	Cancer Genome Interpreter - https://www.cancergenomeinterpreter.org/home
6.	Cancer Driver Log (CanDL) - https://candl.osu.edu/
7.	Precision Medicine KnowledgeBase (PMKB) - https://pmkb.weill.cornell.edu/
8.	MyCancerGenome -  https://www.mycancergenome.org/
9.	COSMIC - http://cancer.sanger.ac.uk/cell_lines
10.	 The Cancer Genome Atlas (TCGA) - https://cancergenome.nih.gov/
11.	International Cancer Genomic Consortium (ICGC) - http://icgc.org/
12.	Drug-Gene Interaction Database (DGIdb) - http://dgidb.genome.wustl.edu/
13.	cBioPortal (integrates various resources, including views on many of the above) - http://www.cbioportal.org/
14.	MolecularMatch (commercial - not sure what is open access for research users) - https://www.molecularmatch.com/
 
 
### Next Steps
1.	Start to ingest from known sources such as ClinVar, CIViC and OncoKB - using data here to inform initial data models and scripts.
2.	Perform a high-level **landscape analysis** of the sources above, and document things like the data types each has to offer, standards and identifiers systems they use, terms of licensing/re-use, data currency and access mechanisms, etc.   I can start this work in a google spreadsheet where anyone can view or contribute. 
3.	Make plan to coordinate data exploration, modeling, parsing/ingest tasks with Wikidata team.
4. Develop use cases and competency questions for this data, with goal of demonstrating what value is added by integrating this knowledge and adding ontological support for query and inference.
